docs/blocks/creating-dynamic-blocks.md

```diff
@@ -116,3 +116,69 @@ There are a few things to notice:
 * The edit function still shows a representation of the block in the editor's context (this could be very different from the rendered version, it's up to the block's author)
 * The save function just returns null because the rendering is performed server-side.
 * The server-side rendering is a function taking the block attributes as an argument and returning the markup (quite similar to shortcodes)
+
+## Live rendering in Gutenberg editor
+
+Gutenberg 2.8 added the [`<ServerSideRender>`](https://github.com/WordPress/gutenberg/tree/master/components/server-side-render) block which enables all the rendering to take place on the server using PHP rather than in JavaScript.
```

Comment:
I think this should make note of the fact that the `ServerSideRenderer` is going to be discouraged in favor of using client-side rendering in JS when possible due to the improved user experience. See https://github.com/WordPress/gutenberg/blob/master/components/server-side-render/README.md

Reply:
Good point, updated in 4e8e8be34cb159de85260adef1e748c31135b2a5